views/layout.pug

```diff
@@ -1,10 +1,25 @@
 doctype html
 html
   head
-    title= title
     link(href='http://fonts.googleapis.com/css?family=Pacifico' rel='stylesheet' type='text/css')
     link(rel='stylesheet', href='/stylesheets/style.css')
+    link(href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css')
+    div.nav
+     div.nav-header-left
+       h1.logo= logo
+       h2.title= title
+
+     div.nav-header-right
+       a(href= '/product') Product
+       a(href= '/users/login') Login
+       input(type="text", placeholder="Email address")
+       button(type="submit" ) Free Trial
+
+
+
   body
     block content
   script(src="/javascripts/index.js")
```

Comment:
Script tags should come after all HTML tags in the body tag (this is before the `div.footer-nav`